src/Tizen.Multimedia.StreamRecorder/StreamRecorder/StreamRecorder.cs

```diff
@@ -34,6 +34,12 @@ public partial class StreamRecorder : IDisposable
         private bool _audioEnabled;
         private bool _videoEnabled;
         private StreamRecorderVideoFormat _sourceFormat;
+        private const string Feature = "http://tizen.org/feature/network.streamrecorder.record";
+
+        private static bool IsSupported()
+        {
+            return System.Information.TryGetValue(Feature, out bool isSupported) ? isSupported : false;
```

Comment:
You can do the same thing just as below code.

```suggestion
            return System.Information.TryGetValue(Feature, out bool isSupported) && isSupported;
```

Reply:
Thank you, i will take

Reply:
hskwon is right.

truth table can reveal it.

ternary operator way.
| return \ isSupported | true | false |
| --- | --- | --- |
|  true |  true | false |
| false | false | false |

&& way
| return \ isSupported | true | false |
| --- | --- | --- |
|  true |  true | false |
| false | false | false |

Reply:
> It is out bool, Your suggestion always TRUE in my opinion

I didn't check your comment early, and I don't understand why you thought it's always true.